comparisons/compareProducts.sh

```diff
@@ -45,7 +45,7 @@ case `basename $fA` in
 
     rm ${os} ${ns}
   ;;
-  step*.txt)
+  *step*.txt)
```

Comment:
is this change on purpose?

Reply:
Yes. The old file has the IB name prepended to it.

Reply:
I can change that.

Reply:
no, I was not sure if it is on purpose or just a typo :-)